### PR TITLE
R4 [Account] Read/Search update to support charge-account and insurance-benefit account types

### PR DESCRIPTION
**Pull request by Coffeevee**
Description
----
Updating the Account resource to contain information for charge-account and insurance-benefit account types

PR Checklist
----
- [x] Screenshot(s) of changes attached before changes merged.
- [ ] Screenshot(s) of changes attached after changes merged and published.

Screenshots Before Merge
----
Screen Shot 2021-06-11 at 3 49 04 PM
Screen Shot 2021-06-08 at 4 13 26 PM

### CODE REVIEW

File: content/millennium/r4/financial/account.md [hunk 1]
```diff
@@ -19,8 +19,10 @@ The Account resource acts as a central record against which charges, payments, a
 
 The account resource supports multiple account types which are defined below.
 
+* `charge-group` - Charge group.
```
Comment (Coffeevee) [inline content/millennium/r4/financial/account.md:22]: We are still in discussion about what to write about the `charge-group`.

**Comment by Coffeevee** [comment]
The second file on this PR is the `.png` file displaying the account relationships diagram.

File: content/millennium/r4/financial/account.md [hunk 2]
```diff
@@ -64,7 +66,7 @@ Search for Accounts that meet supplied query parameters:
 
 _Implementation Notes_
 
-* The Account balance extension is only returned on statement and guarantor-balance types
+* The Account balance extension is only returned on statement, guarantor-balance, and insurance-benefit types
```
Comment (Coffeevee) [inline content/millennium/r4/financial/account.md:69]: I'm wondering if we want to update this line further to include the hotlink to the `#custom-extensions` that lists the `[Account Balance]` here. Thoughts?

Comment (Coffeevee) [reply content/millennium/r4/financial/account.md]: Updated - https://github.com/cerner/fhir.cerner.com/pull/664/commits/0a140bceb0871cf29cb01a44e0889381ff16109f#diff-c8eb47813f9124e188b80dbfe03423839685e5b0f517fa4bcf1a8222a4db0570R69

Screenshots are also updated above.

File: content/millennium/r4/financial/account.md (hunk 1 above)
Comment (alexbeckner1) [reply content/millennium/r4/financial/account.md]: Thanks for making the change. I was thinking the same thing as far as them not being full sentences.  I'm fine either way as long as they are  all there or all gone.

File: content/millennium/r4/financial/account.md (hunk 2 above)
Comment (alexbeckner1) [reply content/millennium/r4/financial/account.md]: +1 to changes.

File: content/millennium/r4/financial/account.md (hunk 1 above)
Comment (Coffeevee) [reply content/millennium/r4/financial/account.md]: Updated - https://github.com/cerner/fhir.cerner.com/pull/664/commits/2f7802952ccfdee91802cf56a58579f52868e24f#diff-c8eb47813f9124e188b80dbfe03423839685e5b0f517fa4bcf1a8222a4db0570R22

**Review by exopsRamba — APPROVED** [review]

File: content/millennium/r4/financial/account.md (hunk 1 above)
Comment (alexbeckner1) [reply content/millennium/r4/financial/account.md]: I heard back from CPA and here is what they came up with "A grouping of charges for billing"

Comment (Coffeevee) [reply content/millennium/r4/financial/account.md]: Updated - https://github.com/cerner/fhir.cerner.com/pull/664/commits/d4ed7ba6b41eae54afb2ac61606e5c9098c664a5#diff-c8eb47813f9124e188b80dbfe03423839685e5b0f517fa4bcf1a8222a4db0570R22

Comment (alexbeckner1) [reply content/millennium/r4/financial/account.md]: +1

**Review by alexbeckner1 — APPROVED** [review]

**Review by MadhurRaj — APPROVED** [review]

**Comment by Zonglin123** [comment]
Screen Shot 2021-06-18 at 12 50 03 PM
Screen Shot 2021-06-18 at 12 50 19 PM

**Pull request merged**